Library/Homebrew/utils.rb

```diff
@@ -121,6 +121,7 @@ def ohai_stdout_or_stderr(message, *sput)
   end
 
   def puts_stdout_or_stderr(*message)
+    message = "\n" if message.empty?
```

Comment:
Doesn't `puts` do this for you already?

Reply:
Something to do with `*message` prevents that working properly - at least in my testing. Basically `message` will be `[]` and `puts([])` doesn't produce a new line.

Reply:
🆒